prime-router/src/test/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaConverterTests.kt

```diff
@@ -0,0 +1,331 @@
+package gov.cdc.prime.router.fhirengine.translation.hl7.schema
```

Comment:
There is no ConfigSchemeConverter class. The rule is that the name of the test class matches the real class. This is also an indication you need to move the tested code into separate files.

Reply:
Separated out in latest commit